--- a/build/fbcode_builder/manifests/cmake
+++ b/build/fbcode_builder/manifests/cmake
@@ -23,8 +23,8 @@ url = https://github.com/Kitware/CMake/releases/download/v3.20.2/cmake-3.20.2-ma
 sha256 = 0100663380a3bd977b001183cd487412db7aad9de6859927bde97e1e6e44e645
 
 [download.any(os=linux,os=freebsd)]
-url = https://github.com/Kitware/CMake/releases/download/v3.20.2/cmake-3.20.2.tar.gz
-sha256 = aecf6ecb975179eb3bb6a4a50cae192d41e92b9372b02300f9e8f1d5f559544e
+url = https://github.com/Kitware/CMake/releases/download/v3.22.0/cmake-3.22.0.tar.gz
+sha256 = 932c038361dc8d9701ee93cb029a877bc0ec3cc1999da4006d006bcbdd748c9
 
 [build.os=windows]
 builder = nop
@@ -40,7 +40,7 @@ CMake.app/Contents/share = share
 
 [build.any(os=linux,os=freebsd)]
 builder = cmakebootstrap
-subdir = cmake-3.20.2
+subdir = cmake-3.22.0
 
 [make.install_args.any(os=linux,os=freebsd)]
 install
